### PR TITLE
Started feed_me

### PR DESCRIPTION
**Pull request by peterkaloyannis**
Started the feed_me project!

### CODE REVIEW

File: feed_me/feed_me/task_manager.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        # Get the score array of the tasks.
+        score_array = self.get_task_score_array(current_time=current_time)
+
+        # Argsort to get the indices of the tasks with the highest score.
+        # TODO: This is a fixed cost approach to tasks, there is potential
+        # TODO: for a linear system solve to find the optimal set of tasks
+        # TODO: in a time window tha maximizes the score completion.
+        max_sorted_indices = np.argsort(score_array)[::-1]
+
+        # Loop through the sorted indices, returning the n_tasks
```
Comment (mnavarrocastillo) [inline feed_me/feed_me/task_manager.py]: i dont like you

File: .gitignore [hunk 2]
```diff
@@ -1,162 +1,10 @@
...
-__pycache__/
-*.py[cod]
-*$py.class
+# General Cleanliness
+.vscode
 
-# C extensions
-*.so
+# Linters
+.ruff_cache
```
Comment (mnavarrocastillo) [inline .gitignore:7]: why?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,26 @@
-# side_projects
-This repository is a hodgepodge monorepo of extremely small project.
+# `side_projects`
+
+This repository is a hodgepodge monorepo of small projects being 
+undertaken by Peter Kaloyannis and Maria Fernanda Navarro Castillo.
```
Comment (mnavarrocastillo) [inline README.md]: Change my name to Maria Navarro

File: README.md [hunk 4]
```diff
@@ -1,2 +1,26 @@
...
+
+## Getting Started
+
+This repository is built upon the pupular [UV](https://github.com/astral-sh/uv)
```
Comment (mnavarrocastillo) [inline README.md]: popular

File: feed_me/feed_me/task_manager.py [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+# The smallest time between the creation time and deadline
+# of a task.
+SMALL_DT = timedelta(seconds=0.1)
+
+class Priority(Enum):
+    """
+    Enumerator representing priority levels.
+
+    These values set the score of a task at the
+    deadline. They also form the basis of
```
Comment (mnavarrocastillo) [inline feed_me/feed_me/task_manager.py]: the basis of?

**Review by mnavarrocastillo — CHANGES_REQUESTED** [review]
I request changes
Also, please help me to set this up and edit in VS

File: README.md (hunk 3 above)
Comment (pkaloyannis-cfs) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (pkaloyannis-cfs) [reply README.md]: Nice catch. Done

**Comment by peterkaloyannis** [comment]
> I request changes Also, please help me to set this up and edit in VS

If I have done my job writing the README.md properly, you should not need assistance. I added a bit more to it so that you could see it.

File: .gitignore (hunk 2 above)
Comment (pkaloyannis-cfs) [reply .gitignore:7]: I deleted most of the other stuff, which was created by github by default. Having a bigger file is just more code to maintain, and so its usually better to build the .gitignore file incrementally with your codebase.

The specific things you are asking about are some temporary files created by vscode and by python when you run it. They are not real code, just useful things your computer uses when you are running scripts.

File: feed_me/feed_me/task_manager.py (hunk 5 above)
Comment (peterkaloyannis) [reply feed_me/feed_me/task_manager.py]: nice catch. fixed.

**Review by mnavarrocastillo — APPROVED** [review]
Good job

**Pull request merged**